src/common/modules/UnicodeTransformationHandler.js

```diff
@@ -110,12 +111,13 @@ function toggleCase(atext) {
     let output = "";
 
     for (let letter of atext) {
-        const upper = letter.toLocaleUpperCase();
-        const lower = letter.toLocaleLowerCase();
-        if (letter === lower && letter !== upper) {
-            letter = upper;
-        } else if (letter === upper && letter !== lower) {
-            letter = lower;
+        // \p{Changes_When_Uppercased}
```

Comment:
I’m sorry, but I still don’t get it. Are these abbreviations?

If so, can’t we write them completely or what is preventing us here?

Reply:
See above. `\p{CWU}` is short for `\p{Changes_When_Uppercased}`.